docs/running-on-kubernetes.md

```diff
@@ -351,6 +351,13 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
+  <td><code>spark.kubernetes.driver.pod.name</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    Name of the driver pod
+  </td>
```

Comment:
Maybe add this: `If not set, the driver pod name would be "spark.app.name" suffixed by a timestamp to avoid confliction with other spark apps.`